grid.css

```diff
@@ -0,0 +1,84 @@
+
+.container {
+   margin-right: auto;
+  margin-left: auto;
+ 
+}
+
+.container:after {
+    content: '\a';
```

Comment:
What u need here is clear:both

Refer this. http://www.w3schools.com/css/css_float.asp